lib/ceych.js

```diff
@@ -83,4 +84,12 @@ Ceych.prototype.disableCache = function() {
   this.cache.stop();
 };
 
+Ceych.prototype.invalidate = function(fn) {
+  const args = Array.from(arguments);
+  const cb = args.pop();
```

Comment:
Probably a programming error if the callback is absent, so do we need to assert for that? 

`memorise` uses `getFunctionCallback` for this reason.